addons/mail/static/src/owl/components/composer/composer.scss

```diff
@@ -209,6 +240,11 @@
     border-top: 1px solid lighten(gray('400'), 5%);
     border-bottom: 1px solid lighten(gray('400'), 5%);
 
+    &:not(.o-composer-is-compact) {
+        border-bottom: 0;
+        border-radius: initial;
```

Comment:
FYI, this style is not applied, because border-radius on `.o_Composer.o-has-current-partner-avatar .o_Composer_buttons > *:last-child` has more specificity.

I think you can ignore this. First, it changes nothing... And second, composer text input will change and the CSS selector above could be adapted to have shortest specificity afterwards

Reply:
Alright !